### DISCUSSION TITLE
deployment/install

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#798] [Announcements]
# deployment/install

设备要求 Windows系统版本最低Windows10+/Windows Server 2019+ Linux系统要支持Python3.10+，推荐Ubuntu 20.04+/CentOS 8+ CPU: 至少1vCPU 内存: Bot无其他插件会占用100MB，其他插件占用视具体插件而定，建议1GB以上 硬盘: 至少1GB空间 GPU: 原生轻雪无需...

https://bot.liteyuki.icu/deployment/install.html

**Comment by snowykami** [comment]
很好用，安装很方便，孩子很喜欢

**Comment by Liteyuki** [comment]
我也喜欢
